app/Relations/HasAlbumThumb.php

```diff
@@ -187,7 +188,7 @@ public function addEagerConstraints(array $models): void
 			});
 		}
 
-		$userID = AccessControl::is_logged_in() ? AccessControl::id() : null;
+		$userID = Auth::id();
```

Comment:
If we simply do

```suggestion
		$user = Auth::user();
```

here, then ... (to be continued below)
```

Reply:
fixed.